.gitignore

```diff
@@ -3,4 +3,9 @@ node_modules
 dist
 *.log
 .env
-.DS_Store
\ No newline at end of file
+.DS_Store
+# Server certificates and keys
+src/server/locals/*.cer
```

Comment:
@maxpluez You need to have a .gitkeep file in these folders or else the folder wouldn't be in github. See https://stackoverflow.com/a/7229996/6001

Reply:
.gitkeep is created